vuepress/docs/.vuepress/components/SpecialLayout.vue

```diff
@@ -235,10 +235,9 @@
             <div style="display:none">
               <p>Install the Entando CLI</p>
               <div class="instruction">
-                curl "https://raw.githubusercontent.com/entando/entando-cli/develop/hr/auto" | ENTANDO_RELEASE="v6.3.0-sprint12" ENTANDO_CLI_VERSION="develop" bash
+                curl -sfL https://get.entando.org | bash
```

Comment:
Command probably isn't going to work as-is for Mac. We can merge but i expect updates.

Reply:
Yep. It's almost identical to what k3s uses so I'm hoping it's close...